tests/lib/passwordChange.js

```diff
@@ -167,7 +162,7 @@ define([
           })
           .then(function (keys) {
 
-            return respond(client._passwordChangeFinish(email, newPassword, oldCreds, keys, sessionToken, { keys: true }), RequestMocks.passwordChangeFinishKeys);
+            return respond(client._passwordChangeFinish(email, newPassword, oldCreds, keys, { keys: true, sessionToken: sessionToken }), RequestMocks.passwordChangeFinishKeys);
```

Comment:
Something to note, in the scenario where keys are requested and no sessionToken is passed, the sessionToken returned will be unverified.